ci/azure-pipeline-connectors-cosmosdb.yaml

```diff
@@ -1,11 +1,30 @@
 trigger:
   branches:
     include:
-      - main
+    - main
   paths:
+    exclude:
+    - /*
     include:
-      - config/*
-      - Connectors/src/CosmosDb/*
+    - behave*
+    - Pipfile*
+    - pyenv.pkgs
+    - '**/*.py'
+    - ci/templates/*.yaml
+    - ci/azure-pipeline-connectors-cosmosdb.yaml
+    - Connectors/src/CosmosDb
```

Comment:
Does this match changes in subdirectories as well?

Reply:
To some limited depth, yes. I tested with controllers in all the commits [here](https://github.com/SteeltoeOSS/Samples/pull/361/commits) unfortunately, it looks like not all of the job history is there, but the only place I had an issue was with the configuration sample, when the path filter was at the component level rather than project level... At this point, all of the filters are at the project level, and changes were tested in project/controller and triggered fine

Reply:
It doesn't work, see https://github.com/SteeltoeOSS/Samples/pull/378.

Reply:
My mistake, I tested the wrong file. It does work:

![image](https://github.com/user-attachments/assets/585a06a2-6b65-4f3a-a63d-789c8a98f47b)